frontend/src/metabase/admin/performance/utils.tsx

```diff
@@ -315,3 +315,13 @@ export const translateConfigFromAPI = (config: CacheConfig): CacheConfig =>
 /** Translate a config from the frontend's format into the API's preferred format */
 export const translateConfigToAPI = (config: CacheConfig): CacheConfig =>
   translateConfig(config, "toAPI");
+
+export const getDefaultValueForField = (
+  strategyType: CacheStrategyType,
+  fieldName?: string,
+) => {
+  const schema = getStrategyValidationSchema(
+    PLUGIN_CACHING.strategies[strategyType],
+  );
+  return fieldName ? schema.cast({})[fieldName] : "";
+};
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +328]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Moved this function here from `frontend/src/metabase/admin/performance/components/StrategyForm.tsx`<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/metabase/metabase/46543?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->